src/index.html

```diff
@@ -15,8 +15,108 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#header"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo-Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#header"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#header"
+            >
+            Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#header"
+            >
+            Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#header"
+              data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#header"
+            >
+            Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#header"
+            >
+            Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#header"
+            >
+            Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#header"
+            >
+            Video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. For example, there should be an empty line before and after each `<li class="nav__item">` block.